lib/hutch/config.rb

```diff
@@ -13,6 +13,7 @@ def self.initialize(params={})
         mq_host: 'localhost',
         mq_port: 5672,
         mq_exchange: 'hutch',  # TODO: should this be required?
+        mq_exchange_options: {},
```

Comment:
I thought about that but then there might be people who directly set `Hutch::Config.set :mq_exchange_options, custom: :thing` and then mistakenly lose the default durable option